openshift-s2i/s2i.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: s2i
+spec:
+  inputs:
+    resources:
+     - name: git-source
+       type: git
+    params:
+    - name: BUILDER_IMAGE
+      description: The location of the builder image.
+      default: quay.io/openshift-pipeline/kbaig_s2i
+    - name: S2I_IMAGE
+      description: The location of the s2i builder image.
+      default: docker.io/centos/nodejs-8-centos7
```

Comment:
Let's provide a `s2i` tasks that has no default, and additional tasks (`s2i-nodejs`, `s2i-python`, …) that uses those images (`centos/nodejs-…`) — That will allow better configuration for each (as an example : I could want nodejs-10 with centos7, or nodejs-8 with something else, so for node, we could have a version parameter or something like that)

Reply:
Will send another pr with ruby, python and nodejs tasks.